ios/Runner/Assets.xcassets/LaunchBackground.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
+  "colors" : [
+    {
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "0.941",
+          "green" : "0.941",
+          "red" : "0.941"
```

Comment:
nit: Ah I think we can use a hex representation, to make it easier to compare with `DesignVariables.mainBackground`?

Like in zulip-mobile:

https://github.com/zulip/zulip-mobile/blob/eb8505c4a0ccfb7bdd817ca9022c11ce57ab69d8/ios/ZulipMobile/Assets.xcassets/Brand.colorset/Contents.json#L8-L10

```
          "blue" : "0xFE",
          "green" : "0x92",
          "red" : "0x64"
```

Reply:
Hopefully that would be an NFC change; hopefully it wasn't a factor in my failing to get things to work (as I mentioned in discussion)

Reply:
Updated.